cmd/cluster-agent/app/servicemapperdigest.go

```diff
@@ -0,0 +1,105 @@
+package app
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	log "github.com/cihub/seelog"
+	"github.com/spf13/cobra"
+
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/status"
+)
+
+func init() {
+	ClusterAgentCmd.AddCommand(svcMapperCmd)
+	svcMapperCmd.SetArgs([]string{"caseID"})
+}
+
+// TODO add all registered nodes command
+var svcMapperCmd = &cobra.Command{
+	Use:   "svcmap [nodeName]",
+	Short: "Print the map between the services and the pods behind them",
+	Long:  ``,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		configFound := false
+		// a path to the folder containing the config file was passed
+		if len(confPath) != 0 {
+			// we'll search for a config file named `datadog-cluster.yaml`
+			config.Datadog.AddConfigPath(confPath)
+			confErr := config.Datadog.ReadInConfig()
+			if confErr != nil {
+				log.Error(confErr)
+			} else {
+				configFound = true
+			}
+		}
+		if !configFound {
+			log.Debugf("Config read from env variables")
+		}
+		nodeName := ""
+		// "*" Can be used to output all nodes.
+		if len(args) == 0 {
+			log.Infof("You need to specify on which node you want to run the service mapper.")
+			return nil
+		}
+		nodeName = args[0]
+		err := getServiceMap(nodeName) // if nodeName == "*", call all.
```

Comment:
Using the wildcard in a command line is cumbersome, you need to escape it...
It's not done for this purpose but to easily glob file system items.